src/pages/CommitDetailPage/CommitCoverage/UploadsCard/useUploads/useUploads.js

```diff
@@ -1,24 +1,22 @@
 import { useParams } from 'react-router-dom'
 
 import { useCommit } from 'services/commit'
-import { useRepoSettings } from 'services/repo'
+import { useRepoOverview } from 'services/repo'
 import { TierNames, useTier } from 'services/tier'
 import { useFlags } from 'shared/featureFlags'
 import { extractUploads } from 'shared/utils/extractUploads'
 
 export function useUploads() {
   const { provider, owner, repo, commit } = useParams()
-  const { data: settings } = useRepoSettings()
+  const { data: overview } = useRepoOverview({ provider, owner, repo })
   const { multipleTiers } = useFlags({
     multipleTiers: false,
   })
   const { data: tierData } = useTier({ provider, owner })
 
   // TODO: We need backend functionality to properly manage access to carryforward flags for team tier members
   const isTeamPlan =
-    multipleTiers &&
-    tierData === TierNames.TEAM &&
-    settings?.repository?.private
+    multipleTiers && tierData === TierNames.TEAM && overview?.private
```

Comment:
Should this be overview.repository.private?

Reply:
Nope, overview doesn't have a repository field on it, we bring the fields right to the top.